src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import Thunk from 'redux-thunk';
+import CityWeather from './cities/cities';
+
+const rootReducer = combineReducers({
+  cityWeathers: CityWeather,
+});
+
+const store = createStore(rootReducer, applyMiddleware(Thunk));
```

Comment:
- [ ] OPTIONAL: Redux logger is a golden source of information and usefulness when trying to debug redux.  I highly recommend you add it.